resources/monitoring/templates/prometheus/additionalPrometheusRules.yaml

```diff
@@ -1,19 +1,40 @@
-{{- if .Values.additionalPrometheusRules }}
-{{- range $name, $config := .Values.additionalPrometheusRules | fromYaml }}
----
-apiVersion: monitoring.coreos.com/v1
-kind: PrometheusRule
-metadata:
-  name: {{ template "prometheus-operator.name" $ }}-{{ $name }}
-  namespace: {{ $.Release.Namespace }}
-  labels:
-    app: {{ template "prometheus-operator.name" $ -}}
-{{ include "prometheus-operator.labels" $ | indent 4 -}}
-{{ if $config.additionalLabels }}
-{{ toYaml $config.additionalLabels | trim | indent 4 -}}
+{{- if or .Values.additionalPrometheusRules .Values.additionalPrometheusRulesMap}}
+apiVersion: v1
```

Comment:
There are some changes done by SRE in this file. We should check with them if the new file works, otherwise we should keep their changes.